db/seeds.rb

```diff
@@ -1,4 +1,5 @@
 jcasimir = User.create(username: "jcasimir", password: "test")
+admin = User.create(username: "admin", password: "admin", role: 1)
```

Comment:
Thankfully this is just a student project, but be careful with this. We're essentially making a public admin account's sign in credentials visible to anyone who views the repo.

Perhaps Figaro could solve this? Not sure.